common/models/update_load_balancer_request.go

```diff
@@ -0,0 +1,38 @@
+package models
+
+import (
+	"fmt"
+
+	swagger "github.com/zpatrick/go-plugin-swagger"
+)
+
+type UpdateLoadBalancerRequest struct {
+	LoadBalancerID string
+	Ports          *[]Port     `json:"ports"`
+	HealthCheck    HealthCheck `json:"health_check"`
+}
+
+func (u UpdateLoadBalancerRequest) Validate() error {
+	if u.LoadBalancerID == "" {
+		return fmt.Errorf("LoadBalancerID is required")
+	}
+
```

Comment:
```
if u.Ports != nil {
    for _, port := range u.Ports {
        if err := port.Validate(); err != nil {
            return err
        }
    }
}

if u.HealthCheck != nil {
    if err := u.HealthCheck.Validate(); err != nil {
        return err
    }
}
```

(if those sub-`Validate()` functions don't exist yet, we should add them)

Reply:
These sub-`Validate()` functions do exist, I checked the respective models.

Reply:
Just a quick note, I had to dereference u.Ports, but otherwise, I used this block verbatim.